static/js/compose.js

```diff
@@ -1287,8 +1287,26 @@ export function initialize() {
             available_providers.big_blue_button &&
             page_params.realm_video_chat_provider === available_providers.big_blue_button.id
         ) {
+            let meeting_name = "";
+            if (compose_state.get_message_type() === "stream") {
+                meeting_name =
+                    compose_state.stream_name() + " > " + compose_state.topic() + " Meeting";
```

Comment:
I think if the `topic` input is empty, we probably want this to not include the `> {topic}` part, since `stream_name >  Meeting` looks like a bug.

Reply:
I asked a question about this in #frontend